vms/platformvm/validators/manager.go

```diff
@@ -413,5 +415,42 @@ func (m *manager) OnAcceptedBlockID(blkID ids.ID) {
 }
 
 func (m *manager) GetCurrentValidatorSet(ctx context.Context, subnetID ids.ID) (map[ids.ID]*validators.GetCurrentValidatorOutput, uint64, error) {
-	return m.state.GetCurrentValidatorSet(ctx, subnetID)
+	result := make(map[ids.ID]*validators.GetCurrentValidatorOutput)
+	baseStakers, l1Validators, height, err := m.state.GetCurrentValidators(ctx, subnetID)
+	if err != nil {
+		return nil, 0, err
```

Comment:
I checked it's safe to wrap the error

```suggestion
		return nil, 0, fmt.Errorf("failed to get current validators: %w", err)
```